libs/components/.eslintrc.cjs

```diff
@@ -0,0 +1,80 @@
+module.exports = {
+  // automatically extends root eslint-config (which extends @shiftcode/eslint-config-recommended for TS)
+  extends: ['@shiftcode/eslint-config-recommended/ng-config'],
+  overrides: [
+    {
+      plugins: ['@shiftcode/rules'],
+      files: ['*.ts'],
+      rules: {
+        'import/no-internal-modules': ['error', { allow: ['rxjs/operators', '@angular/**/*'] }],
+        '@typescript-eslint/naming-convention': [
+          'error',
+          {
+            selector: 'property',
+            format: ['camelCase', 'UPPER_CASE'],
+            leadingUnderscore: 'allow'
+          },
+          {
+            selector: "objectLiteralProperty",
+            format: null,
+            filter: {
+              regex: '^(\\[.*\\]|\\(.*\\)|aria-.*)$',
+              match: true
+            }
+          },
+        ],
+        '@shiftcode/rules/import-denylist': [
+          'error', {
+            patterns: [
+              /\.\/(models|static|core|shared)\/.*/,
+              /src\/.*/
+            ],
+          },
+        ],
+        '@angular-eslint/directive-selector': [
+          'error',
+          {
+            type: 'attribute',
+            prefix: ['pl', 'ch', 'sc'],
+            style: 'camelCase',
+          },
+        ],
+        '@angular-eslint/component-selector': [
+          'error',
+          {
+            type: 'element',
+            prefix: ['pl', 'ch', 'sc'],
+            style: 'kebab-case',
+          },
+        ],
+      },
```

Comment:
These rules can be removed since they are project specific. I like the denylist that you added though.